Formula/fastlane.rb

```diff
@@ -29,7 +30,7 @@ def install
 
     (bin/"fastlane").write <<~EOS
```

Comment:
I think we can refactor this into a [`write_env_script`](https://rubydoc.brew.sh/Pathname.html#write_env_script-instance_method).

Reply:
https://github.com/Homebrew/homebrew-core/pull/77490

Submitted as a separate PR since it seems like this might be tied up waiting on upstream Ruby 3.0 testing...

Reply:
@LinusU Thank you! It was a big migration to Ruby 3.0 and there have been a few small regressions so far. I'd be fine with this but Homebrew users can't downgrade to previous versions so I'd like to wait a month or two before committing Homebrew to Ruby 3.0 😇